docs/install_guides/pypi-openvino-dev.md

```diff
@@ -73,22 +69,22 @@ On Windows:
 openvino_env\Scripts\activate
 ```
 
-### Step 4. Set Up and Update pip to the Highest Version
+### Step 3. Set Up and Update pip to the Highest Version
```

Comment:
Step 3. Set Up and Update pip to the Highest Version ---> Step 3. Set Up and Update Pip to the Highest Version
all words in headings except articles and prepositions are usually capitalized

Reply:
pip is a recursive acronym for "Pip Installs Packages", so I believe all letters shall be capitalized.